src/packages.ts

```diff
@@ -14,16 +14,21 @@ export class TypeList
     static OTHER = { id: 3, name: 'other', label: 'Other', autobuild: new Array<string>() };
 
     // For internal use only
-    static INVALID = { id: 4, name: undefined, label: undefined, autobuild: new Array<string>() };
-    static CONFIG = { id: 5, name: undefined, label: undefined, autobuild: new Array<string>() };
+    static INVALID = { id: 4, name: 'invalid', label: 'internal', autobuild: new Array<string>() };
+    static CONFIG = { id: 5, name: 'config', label: 'internal', autobuild: new Array<string>() };
```

Comment:
What is the point of defining the label (and the name?) for internal types? If we leave them undefined they get automatically hidden from the status bar (and save us from writing isConfig()/isInvalid() checks there).

Reply:
Multiple:
 - null values in status bar methods is actually not allowed with strict null checks
 - I don't like having the invalid/config packages "magically" ignored in the code paths where they should be ignored. I prefer the explicit check.

Reply:
> null values in status bar methods is actually not allowed with strict null checks

Yep, missed that. We could automatically hide 'internal' then.

> I don't like having the invalid/config packages "magically" ignored in the code paths where they should be ignored. I prefer the explicit check.

This kinda violates the "Tell, don't ask" principle. Also, if we really want to keep test coverage at high levels, we end up having to frequently write test cases for this checks. But again, this is premature optimization because it only happens in one place so let's leave as you suggested.